kinto/core/schema.py

```diff
@@ -0,0 +1,110 @@
+import six
```

Comment:
It might be nice to have a module-level docstring explaining what's supposed to go in this file vs. what goes in `kinto.core.resource.schema`.